Gemfile

```diff
@@ -42,16 +43,14 @@ group :development, :test do
   gem 'capybara-email'
   gem 'guard-rspec', require: false
   gem 'terminal-notifier-guard'
-  gem 'spring-commands-rspec'
-  gem 'spring'
```

Comment:
These were causing errors in trying to load the rails console. I believe they're removed in a later version of rails anyway.

Reply:
👍 I also believe that is true